### PR TITLE
MSVC 2013 x64 build success

### PR DESCRIPTION
**Pull request by ghost**
# What this PR does?
* fix build on Windows MSVC 2013 x64

# Dependencies

* IlmBase library from the OpenEXR project

*Dependencies will be downloaded and built automatically if they are not found on the system.*

# How to build:

* ` git clone --recursive -b feature/windows-msvc-build git@github.com:qubecinema/CTL.git`
* `cd CTL`
* `mkdir build`
* `cd build`
* `cmake -G "Visual Studio 12 2013 Win64" ..`
* `cmake --build . --config release `

### CODE REVIEW

File: unittest/IlmCtl/CMakeLists.txt [hunk 1]
```diff
@@ -57,7 +56,14 @@ file(
...
         testVaryingReturn.ctl
-        testVSArrays.ctl
-    DESTINATION
-        ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
-)
+        testVSArrays.ctl)
+        
+if (MSVC)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Release)
```
Comment (RajaBoseLeoM) [inline unittest/IlmCtl/CMakeLists.txt:63]: Do we need to copy files in both release and debug directories everytime? Instead would it be better to copy in respective directories based on the target cmake build type?

**Review by RajaBoseLeoM — CHANGES_REQUESTED** [review]

File: unittest/IlmCtl/CMakeLists.txt (hunk 1 above)
Comment (ghost) [reply unittest/IlmCtl/CMakeLists.txt:63]: Cmake config doesn't restrict MSVC to build in Release and Debug mode.
One can choose mode directly from MSVC UI. Without copying these test files the IlmCtl test will fail .This is only for testing purpose

Comment (RajaBoseLeoM) [reply unittest/IlmCtl/CMakeLists.txt:63]: As discussed we  need to check if we are able to build successfully in x86 as well

Comment (ghost) [reply unittest/IlmCtl/CMakeLists.txt:63]: @RajaBoseLeoM I've checked, everything is fine.

**Review by RajaBoseLeoM — APPROVED** [review]

**Review by yuriyvolkov — APPROVED** [review]

**Pull request merged**